idlib/ParallelJobList.cpp

```diff
@@ -126,7 +126,7 @@ idParallelJobList_Threads
 ================================================================================================
 */
 
-static idCVar jobs_longJobMicroSec( "jobs_longJobMicroSec", "10000", CVAR_INTEGER, "print a warning for jobs that take more than this number of microseconds" );
+static idCVar jobs_longJobMicroSec( "jobs_longJobMicroSec", "500000", CVAR_INTEGER, "print a warning for jobs that take more than this number of microseconds" );
```

Comment:
That's the question. If I don't silence it, we will get useless spam from the image load process. id probably added that log because it was a good indicator for how they were using the job system in D3 BFG. But currently we're only using it for two things - image loading during level load and adding model surfaces in parallel during frontend frame preparation. The warning is useless for the first use case, it might have merit for the second, I'm not sure. I guess we could either remove it entirely or somehow add an exception for the image load.

Reply:
I think that reflects the generic usage of job system in games: processing tasks which happen between two frames. It is not clear how much of that would happen in TDM. We can leave it as is, but if more jobs appear, then add settings about that in job system interface (aka "realtime" or "offline").

Reply:
I actually found a better way - digging through the code again, id already added an exception to this rule: they have a third type of job queue for "utility" jobs that we haven't been using so far. That one is exempt from the warnings. I modified the image loading code to create its own utility job queue instead of recycling the frontend job list (which wasn't technically correct, anyway), so now the warnings are gone without having to increase the timeout value :)